### PR TITLE
Bump v2.2

### PR DESCRIPTION
**Pull request by ajzkk**

### CODE REVIEW

**Comment by aashishgurung** [comment]
I noticed last releases included `release/omise-eccube.tar.gz`. We might need to do the same.

**Comment by danfowler** [comment]
@ajzkk @aashishgurung 2.2 implies we added features in a backwards-compatible way. What about 2.1.1?

**Comment by ajzkk** [comment]
> 2.2 implies we added features in a backwards-compatible way. What about 2.1.1?

@danfowler  In changelog, it is mention as below
> note, we were changed our versioning structure from [platform-major].[platform-minor].[plugin-major].[plugin-minor] to [platform-major].[plugin-major].
So, this update was made base on EC-Cube 2.x series with second release (the first one is v2.13.0.0 = 2.0). So, this release will release under version 2.1


https://github.com/omise/omise-eccube/blob/master/CHANGELOG.md

**Comment by danfowler** [comment]
OK

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+### v2.2 _(September 5, 2023)_
+- Removed hard coded certificate (PR [#9](https://github.com/omise/omise-eccube/pull/12))
```
Comment (danfowler) [inline CHANGELOG.md]:
```suggestion
- Removed hard-coded certificate (PR [#9](https://github.com/omise/omise-eccube/pull/12))
```

**Comment by ajzkk** [comment]
> I noticed last releases included `release/omise-eccube.tar.gz`. We might need to do the same.

Thanks for notice, I have included.

**Review by aashishgurung — APPROVED** [review]

**Pull request merged**